components/User/data-access/UserDAO.ts

```diff
@@ -5,16 +5,25 @@ export class UserDAO {
 
 	public static async getAllUsers(): Promise<UserModel[]> {
 		return UserModel.findAll({
-			where: {'deletedAt': null},
 			attributes: {exclude: ['password']}
 		})
 			.then(users => users);
 	}
 
-	public static async getUserById(id: number): Promise<UserModel | null> {
+	public static async getUserById(id: string): Promise<UserModel | null> {
 		return UserModel.findOne({
 			where: {
 				id,
+				'deletedAt': null
+			},
+			attributes: {exclude: ['password']}
+		});
+	}
+
+	public static async getUsersByIds(ids: string[]): Promise<UserModel[] | null> {
+		return UserModel.findAll({
+			where: {
+				id: ids,
```

Comment:
Are you using getUsersByIds anywhere?
If yes, I think you need to use`in` sequelize query opeator.  You can find an examples here https://sequelize.readthedocs.io/en/v3/docs/querying/